src/cli/args.ts

```diff
@@ -16,7 +17,8 @@ const cliArgsDescription: { [k: string]: string } = {
 
 interface CLIArgs {
 	help?: boolean;
-	workdir: string;
+	addrepo?: string;
+	workdir?: string;
```

Comment:
Don't make this optional, it will change the current behavior, Flow should be like this if the flag ```--workdir```  is present then we can test its value based on a regex that if it's of a type ```URL``` or not and based on this type you can call the appropriate function.

If we handle it like this, the current behavior of application won't change which is:- 

1) If --workdir flag is not present, then the command ```$ metacall-deploy``` will deploy the current repo is which user is present --> ```process.cwd()```